packages/atomic/src/utils/day-js-locales.ts

```diff
@@ -0,0 +1,43 @@
+import dayjs from 'dayjs';
+
+const locales: Record<string, () => Promise<unknown>> = {
```

Comment:
this automatically adds each files to the build & lazy loads the locales as needed

Reply:
The locales we support won't probably change in the future, but we could always...
- Put this file into .gitignore
- Add a pre-build step to parse the `locales.json` file or the `node_module/dayjs/locale` folder to dynamically add locales